rdocs/00Index.html

```diff
@@ -0,0 +1,484 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
```

Comment:
Lets name this directory rdocs-1.2 or rdocs-master to correspond to the github branch name